src/fabric/test/fabric2_tutil.erl

```diff
@@ -0,0 +1,99 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
```

Comment:
At the risk of bike shedding, `tutil` seems a little terse, and I'd prefer `fabric2_test_util` if it were up for a vote.

Reply:
Changed to fabric2_test_util